packages/math-input/src/components/input/math-wrapper.ts

```diff
@@ -119,9 +119,13 @@ const KeysForJumpContext = {
 };
 
 class MathWrapper {
+    MQ: any;
+    mathQuillInput: any;
+    callbacks: any;
```

Comment:
I might come back and fix these. Right now I'm trying to focus on errors that pop up when I move these to TS.